server/src/models/channels.js

```diff
@@ -5,11 +5,11 @@ const { Schema } = mongoose;
 const { ObjectId } = Schema;
 
 const ChannelSchema = new Schema({
-  admins: [String],
-  channelId: ObjectId,
+  name: String,
+  admins: [ObjectId],
```

Comment:
Any reason why not references used here.

Reply:
Oops this PR was completed on Friday. In Hackathon, schema code is fixed, and now all the code is having references..
Let me rework on PR, and will assign you again.